examples/Makefile

```diff
@@ -129,49 +126,56 @@ clean:
 
 # Don't rely on default rules for the Fortran and Java examples
 
+hello_c: hello_c.c
+	$(MPICC) $(CFLAGS) $(LDFLAGS) $? $(LDLIBS) -o $@
+ring_c: ring_c.c
+	$(MPICC) $(CFLAGS) $(LDFLAGS) $? $(LDLIBS) -o $@
+connectivity_c: connectivity_c.c
+	$(MPICC) $(CFLAGS) $(LDFLAGS) $? $(LDLIBS) -o $@
```

Comment:
I think you missed the C++ targets.

From CI:

```
g++ -g    hello_cxx.cc   -o hello_cxx
hello_cxx.cc:17:17: fatal error: mpi.h: No such file or directory
 #include "mpi.h"
                 ^
compilation terminated.
<builtin>: recipe for target 'hello_cxx' failed
```

Reply:
a good reminder of why we have CI :).  I forgot to enable the C++ bindings when I was testing locally...